seqr/views/utils/individual_utils.py

```diff
@@ -72,22 +71,30 @@ def add_or_update_individuals_and_families(project, individual_records, user):
 
     for record in individual_records:
         _update_from_record(
-            record, user, families_by_id, individual_lookup, updated_families, updated_individuals, parent_updates, updated_notes)
+            record, user, families_by_id, individual_lookup, updated_family_ids, updated_individuals, parent_updates, updated_note_ids)
 
     for update in parent_updates:
         individual = update.pop('individual')
         is_updated = update_individual_from_json(individual, update, user=user)
         if is_updated:
             updated_individuals.add(individual)
             if individual.family.pedigree_image:
-                updated_families.add(individual.family)
+                updated_family_ids.add(individual.family_id)
```

Comment:
its the underlying database id for the family, its how django links the individual to the family at the database level. Its faster to query the individual table for the foreign key rather than do a join on the family table